backends/conrod_glium/src/lib.rs

```diff
@@ -416,6 +416,25 @@ impl GlyphCache {
     }
 }
 
+pub trait Display {
+    fn opengl_version(&self) -> &glium::Version;
+    fn framebuffer_dimensions(&self) -> (u32, u32);
+    fn hidpi_factor(&self) -> f64;
+}
+
+impl Display for glium::Display {
+    fn opengl_version(&self) -> &glium::Version {
+        self.get_opengl_version()
+    }
+
+    fn framebuffer_dimensions(&self) -> (u32, u32) {
```

Comment:
I don't believe there's any way for this trait to have a method named `get_framebuffer_dimensions` (or `get_opengl_version` or `get_hidpi_factor`) which would have been nice for consistency. If I'm mistaken, I'd love to hear it.

Reply:
I figured it out!

Reply:
Nope, nevermind.